x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/engine_nav.scss

```diff
@@ -6,21 +6,15 @@
  */
 
 .appSearchNavEngineLabel {
-  padding-top: $euiSizeS;
+  margin-left: $euiSizeS;
+  padding-top: $euiSizeXS;
   padding-bottom: $euiSizeS;
 
-  .euiText {
-    font-weight: $euiFontWeightMedium;
-  }
   .euiBadge {
     margin-top: $euiSizeXS;
   }
 }
 
-.appSearchNavIcons {
-  // EUI override
-  &.euiFlexItem {
-    flex-grow: 0;
-    flex-direction: row;
-  }
+.appSearchNavIcon {
+  order: 1;
```

Comment:
👀 👀 👀 

Seeing `order` rules in CSS always raises the hairs on the back of my neck. I trust there's a good reason for this, but could you leave a small comment in the code explaining what that reason is?

Reply:
Nice catch on this Byron! Kibana renders it's icons to the left of text instead of to the right:

<img width="158" alt="" src="https://user-images.githubusercontent.com/549407/122792652-378b2200-d26f-11eb-86aa-b800820575f5.png">

But we want the icons to the right, to match our previous UI:

<img src="https://images.zenhubusercontent.com/5e1e00795b295acb424f1897/f5e95d29-371c-47b8-a391-610fff95905f" width="300" alt="">

If it had been more annoying than a simple CSS/flex re-order to do this I might have left them alone, but I figured since there's an easy way to quickly replicate our old nav icon UI/UX I might as well do so.

Reply:
Also, if EUI or Davey pushes back on this / wants it removed, it's definitely easy enough to do so in the future!

Reply:
Oh, just saw the request was for a code comment 🤦‍♀️ hope this looks good!

```suggestion
  // EuiSideNav renders icons to the left of the nav link by default, but we use
  // icons as warning or error states & prefer to render them on the right side of the nav
  order: 1;
```